backend/fs/DimensionFS.cpp

```diff
@@ -425,6 +425,22 @@ std::vector<double> RangeDimensionFS::ticks() const {
     return ticks;
 }
 
+std::vector<double> RangeDimensionFS::ticks(size_t start, size_t count) const {
+    std::vector<double> ticks;
+    /*
+    Group g = redirectGroup();
+    if (g.hasData("ticks")) {
+        g.getData("ticks", ticks);
+        return ticks;
+    } else if (g.hasData("data")) {
+        g.getData("data", ticks);
+        return ticks;
+    } else {
+        throw MissingAttr("ticks");
+    }
+    */ // FIXME
+    return ticks;
```

Comment:
I guess we don't have tests for this yet :)

Reply:
correct :)